platform/core/src/DataSources/IWebApiDataSource.js

```diff
@@ -59,6 +60,7 @@ function create({
     store: store || defaultStore,
     getImageIdsForDisplaySet,
     retrieveSeriesMetadata,
+    deleteStudyMetadataPromise,
```

Comment:
This does need to be exposed :+1:
I wonder if this version has issues where navigating back to the list after loading a route still shows requests as in-flight.

Reply:
Lets look into this at some point.